packages/edit-site/src/components/save-panel/index.js

```diff
@@ -100,6 +101,14 @@ export default function SavePanel() {
 	const { setIsSaveViewOpened } = useDispatch( editSiteStore );
 	const onClose = () => setIsSaveViewOpened( false );
 
+	const saveBtnRef = useRef();
+	useEffect( () => {
```

Comment:
Because it takes too long for the isSaveViewOpen to fully resolve, the Open save panel button is not rendered in time causing a focus loss to occur. Using the effect here prevents this.